pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -40,15 +40,22 @@ const (
 	checkSleepDuration = time.Second
 	diskByIDPath       = "/dev/disk/by-id/"
 	diskSCSIPrefix     = "wwn-0x"
-	diskformat         = "diskformat"
-	datastore          = "datastore"
-	StoragePolicyName  = "storagepolicyname"
+	// diskformat parameter is deprecated as of Kubernetes v1.21.0
```

Comment:
Shouldn't all these deprecations be part of Kubernetes doc as well? Are you planning to address that in a separate PR?

Reply:
This PR is needed for release notes to be generated.

We should submit another PR to update the doc as well: https://kubernetes.io/docs/concepts/storage/volumes/#vsphere-csi-migration